nilearn/plotting/html_surface.py

```diff
@@ -20,26 +20,85 @@ class SurfaceView(HTMLDocument):
     pass
 
 
+def _mix_colormaps(fg, bg):
+    """Mixes foreground and background arrays of RGBA colors.
+
+    Args:
```

Comment:
Can you write the docstring in numpy-style ?

Reply:
Done!